lib/Models/WebMapServiceCatalogItem.js

```diff
@@ -662,18 +829,18 @@ WebMapServiceCatalogItem.prototype.updateFromCapabilities = function(capabilitie
         }
     }
 
-    updateValue(this, overwrite, 'tilingScheme', tilingScheme);
+    updateValue(wmsItem, overwrite, 'tilingScheme', tilingScheme);
 
-    if (!defined(this.parameters)) {
-        this.parameters = {};
+    if (!defined(wmsItem.parameters)) {
+        wmsItem.parameters = {};
     }
-    updateValue(this.parameters, overwrite, 'srs', srs);
+    updateValue(wmsItem.parameters, overwrite, 'srs', srs);
 
-    if (this.isNcWMS) {
-        updateValue(this, overwrite, 'colorScaleMinimum', -50);
-        updateValue(this, overwrite, 'colorScaleMaximum', 50);
+    if (wmsItem.supportsColorScaleRange) {
+        updateValue(wmsItem, overwrite, 'colorScaleMinimum', -50);
+        updateValue(wmsItem, overwrite, 'colorScaleMaximum', 50);
```

Comment:
I just tried adding `"colorScaleMinimum": -25` to the json file, and it gets overridden to -50 - would it be worth letting this be configurable?